index.js

```diff
@@ -0,0 +1,69 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+let button = document.querySelector("button");
+button.addEventListener("click", addReview);
+
+})
+
+let url = "https://ghibliapi.herokuapp.com/films";
+
+
+axios.get(url)
+    .then((response) => {
+        let dropDown = document.getElementById("dropMenu");
+        let movies = response.data;
+        console.log(movies)
+        for (let i of movies) {
+            let option = document.createElement("option");
+            option.text = i.title;
+            option.id = i.release_date;
+            option.value = i.description;
+            dropDown.add(option);         
+        }
+  
+    })
+    .catch((err) => {
+        console.log("Oops, there's an error");
+    })
+
+    
+function switchMovies(movies) {
+        let dropDown = document.getElementById("dropMenu");
+        let selected = dropDown.selectedIndex;
+        // console.log()
+
+
+        let filmInfo = document.getElementById("filmInfo");
+        filmInfo.innerHTML = "";
+    
+        let filmTitle = document.createElement("h3");
+        filmTitle.innerText = dropDown[selected].text;
+        filmInfo.appendChild(filmTitle);
```

Comment:
`index.js:30:switchMovies()`. This was clever! You are using the `option` element itself for storing all the information about he movie, then use `.selectedIndex` to then grab the option in the dropdown at that index to access the option's properties and create the movie 'card' with the title, release date and description. First time seeing something like this. The only part that is less than ideal here is the fact that you are putting the entire film description in the `.value` property of the `option`. Historically when forms were submitted without JavaScript the browser would send the information in the format: `select-name=select-value` and example of that would be `gender=male`. It is standard to use a smaller/shorter value for the option in your case the value of you option would be the entire description for the movie.

Reply:
Besides what I mention above, our intent here was for you to keep the `id` of the movie in the option's `value` property and then make a network request to `/films/<film-id>` to get the details of the movie. Fixing this to make that other net request would bump your grade for standard `EF.10.c: Make requests to APIs` up